tests/test-cases/image/.gitlab-ci.yml

```diff
@@ -8,7 +8,7 @@ default:
 test job:
   extends: .image
   script:
-    - echo "Test something"
+    - echo "Test something ${GCL_PROJECT_DIR_ON_HOST}"
```

Comment:
let's remove this, since we've already tested in `tests/test-cases/predefined-variables/integration.test.ts`